src/docs/Components/Form/Toggle/index.mdx

```diff
@@ -75,6 +75,17 @@ You can use a reactive label, witch will change depending on the state of the to
 
 <Preview path="toggle-states-label" nude />
 
+### Label on two lines
+
+In uncommon use cases where the text is on two lines, the text box must stay centred with the toggle.
+
+<Highlight theme="warning" title="Be aware">
+  Please think well about the toggle use and its label. For remind, a toggle is used to have instant feedback, use a short label is the best way.
```

Comment:
As a reminder, a toggle is used to have instant feedback, using a short label is the best way to use this component.